src/ConfigR/BasicConfig.cs

```diff
@@ -6,11 +6,11 @@ namespace ConfigR
 {
     using System.Collections.Generic;
     using System.Globalization;
-    using Common.Logging;
+    using Logging;
 
     public partial class BasicConfig : ISimpleConfig
     {
-        private static readonly ILog log = LogManager.GetCurrentClassLogger();
+        private static readonly ILog log = LogProvider.For<BasicConfig>();

```

Comment:
For<T> doesn't work with static classes :(

-----Original Message-----
From: "Adam Ralph" <notifications@github.com>
Sent: ‎08/‎10/‎2015 18:10
To: "config-r/config-r" <config-r@noreply.github.com>
Cc: "Christos Matskas" <christos.matskas@softwarelounge.co.uk>
Subject: Re: [config-r] Logging uprade to LibLog (#205)

In src/ConfigR/BasicConfig.cs:
>  
>      public partial class BasicConfig : ISimpleConfig
>      {
> -        private static readonly ILog log = LogManager.GetCurrentClassLogger();
> +        private static readonly ILog log = LogProvider.For<BasicConfig>();
Out of interest, why For<T>() here and GetCurrentClassLogger() elsewhere?
—
Reply to this email directly or view it on GitHub.